spiceaidocs/docs/components/data-connectors/abfs.md

```diff
@@ -133,27 +145,29 @@ datasets:
       file_format: csv
 ```
 
-### Using secrets for container and account name
+### Connecting to the Storage Emulator
 
 ```yaml
 datasets:
-  # dummy_container will be overridden by the value in `abfs_container`
-  - from: abfs://dummy_container/my_csv.csv
-    name: prod_data
+  - from: abfs://test_container/test_csv.csv
+    name: test_data
     params:
-      abfs_account: ${ secrets:PROD_ACCOUNT }
-      abfs_container: ${ secrets:PROD_CONTAINER }
+      abfs_use_emulator: true
       file_format: csv
 ```
 
-### Connecting to the Storage Emulator
+### Using secrets for Account and Container
+
+When using secrets for `abfs_container`, a dummy container name needs to be provided in the `from` field. This dummy value will be replaced by the value in the secret at runtime.
```

Comment:
This is very weird UX, was this reviewed? We should only have one place to define the container name. If people want to specify the container name in a secret, then we perform the secret replacement syntax in the `from` field.

Reply:
This has been fixed as discussed